pkg/agent/flowexporter/types.go

```diff
@@ -41,7 +41,10 @@ type Connection struct {
 	// For established connections: StopTime is latest time when it was polled.
 	StopTime time.Time
 	// IsPresent flag helps in cleaning up connections when they are not in conntrack table anymore.
-	IsPresent          bool
+	IsPresent bool
+	// IsExportedAndDying flag is for marking the dying connection while flow records have been
+	// exported.
+	IsExportedAndDying bool
```

Comment:
Can we use `IsTerminatedAndExported` instead to avoid confusion with conntrack status flag `dying`? And else where too where dying is used to represent this state.

Suggestion for comment:
"IsTerminatedAndExported flag is to indicate that the connection is about to be removed from the conntrack table and the corresponding flow record is exported."